proto/executor.proto

```diff
@@ -292,8 +294,20 @@ message GroupingSet {
 	repeated GroupingExpr grouping_exprs = 1;
 }
 
+// Deprecated in the nearly feature usage
 message Expand{
 	repeated GroupingSet grouping_sets = 1; // for grouping sets like: expr[a,b] and expr[c]
 	optional Executor child = 2;
 	optional uint64 version = 3;            // expand version
 }
+
+message ProjectExpr {
+	repeated Expr exprs = 1;
+}
+
+message Expand2{
```

Comment:
The `Expand2` with the `version` as its member variable is a little strange. Do we really need the `version` since we've split it?

I think we can first remove it, to see whether we need it later.